lib/cocoapods/sources_manager.rb

```diff
@@ -88,13 +88,18 @@ def update(source_name = nil, show_output = false)
         end
 
         changed_spec_paths = {}
+        # Ceate the Spec_Lock file if needed and lock it so that concurrent
+        # repo updates do not cause each other to fail
+        f = File.open("#{Config.instance.repos_dir}/Spec_Lock", File::CREAT)
+        f.flock(File::LOCK_EX)
```

Comment:
do we need to also unlock the file?

Reply:
The file should be unlocked once it gets closed, right?

Reply:
I',m not sure!

Reply:
The tests wouldn’t be passing if closing didn’t unlock the file so we’re good